register.html

```diff
@@ -20,15 +20,47 @@ <h1 class="logo">Relaxr</h1>
 					<ul>
 						<li><a href="index.html">Home</a></li>
 						<li><a href="blog.html">Blog</a></li>
-						<li><a href="#">Contact</a></li>
+						<li><a href="contact.html">Contact</a></li>
 					</ul>
 				</nav>
 			</div>
 		</header>
 		<section id="main">
 			<div class="container">
 				<section id="contact">
+					<form novalidate id="registrationForm">
+					<p>What's your primary intended use?</p>
+					<label for="usage">
+						Business
+					</label>		
+						<input type="radio" name="usage" id="usage" value="business">
+					<label for="usage">
+						Personal
+					</label>	
+						<input type="radio" id="personal" name="usage" value="personal">
+					<p>
+						What most interests you about Relaxr? Select all that apply
+					</p>
+					<label for="checkbox">Schedule meetings for you</label>
+						<input type="checkbox" name="interests" id="interests" value="">
+					<label for="checkbox">Excel automation to complete your work for you
+						<input type="checkbox" name="interests" id="interests" value="">
```

Comment:
these inputs can all share the `name` `"interests"`, but they must have unique `id` values. The corresponding `label`'s `for` attribute should equal the `id`